client/src/components/RecipeSearch.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react"
+import SearchedRecipeTile from "./SearchedRecipeTile"
+
+const RecipeSearch = (props) => {
+  const [formData, setFormData] = useState({
+    searchQuery: ""
+  })
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const [searchResults, setSearchResults] = useState(undefined)
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    try {
+      const response = await fetch("/api/v1/recipe-search", {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(formData)
+      })
+      if(response.ok) {
+        const body = await response.json()
+        setSearchResults(body.searchResults)
+      }
+    } catch(err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
+  const handlePageClick = async (event) => {
+    event.preventDefault()
+    const targetPage = event.currentTarget.getAttribute("value")
+    const pageData = {
+      targetPage,
+      searchQuery: searchResults.searchQuery
+    }
+    try {
+      const response = await fetch("/api/v1/recipe-search/next", {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json"
+        }),
+        body: JSON.stringify(pageData)
+      })
```

Comment:
you can actually REUSE your original search fetch request IF you refactor it to use query params, so that you dont need to make this odd one.
Here is how we could follow convention with get
```
const response = await fetch(`/api/v1/recipe-search?query=${pageData.searchQuery}&targetPage=${pageData.targetPage}`
```
this would be more conventional, and much dryer